src/lib/config.ts

```diff
@@ -36,5 +36,5 @@ export class FaConfig {
    *
    * @default 'unset'
```

Comment:
Please update the default value in JSDoc comment accordingly.

Reply:
Of course.